src/middlewares/validator.js

```diff
@@ -0,0 +1,20 @@
+import Joi from '@hapi/joi';
+import response from '../utils/response';
+
+export default (schema) => (req, res, next) => {
+  if (!schema) return next();
+
+  const { body, params, query } = req;
+
+  Joi.validate({ ...body, ...params, ...query }, schema, {
+    abortEarly: false,
+    stripUnknown: true
```

Comment:
Add `allowUnknown: true` to this line too.

Reply:
Okay, thank you